tensorrt_llm/serve/cluster_storage.py

```
@@ -590,7 +590,7 @@ async def set(self,
     @handle_etcd_error(return_on_error=None)
     async def get(self, key: str) -> str:
         data, meta = self.client.get(key)
```

Ruff (0.16.1)

[warning] Unpacked variable `meta` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)